src/lib/content.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { createRoot } from 'react-dom/client'
+import { mathifyElement } from './utils'
+import { ContentLoader } from '../components/ContentLoader'
+
+const OS_RAISE_CONTENT_CLASS = 'os-raise-content'
+
+export const renderContentElements = async (): Promise<void> => {
+  const osContentItems = document.querySelectorAll(`.${OS_RAISE_CONTENT_CLASS}`)
+  const contentPromises: Array<Promise<void>> = []
+
+  osContentItems.forEach((elem) => {
+    const htmlElem = elem as HTMLElement
+    const contentId = htmlElem.dataset.contentId
+
+    if (contentId === undefined) {
+      console.log('WARNING: Ignoring os-raise-content with missing UUID data attribute')
+      return
+    }
+
+    if (htmlElem.hasChildNodes()) {
+      console.log('WARNING: Found non-empty os-raise-content')
+    }
+
+    mathifyElement(elem).catch((error) => {
+      console.error(error)
+    })
```

Comment:
There is some background in [this comment](https://github.com/openstax/k12/issues/64#issuecomment-1073916105), but the general gist is that we are completely outside of Moodle's MathJax filter (e.g. it will do nothing for us due to some of our findings in that issue thread). However, we go ahead and "patch" MathJax into the `window` using Moodle's configuration if we find it (this is what `loadMathJax` is for). The `mathifyElement` is abstracting `MathJax` a bit, but it's implementation of `window.MathJax.Hub.Queue` is generally how you tell MathJax to render something using [Queue()](https://docs.mathjax.org/en/v2.4-latest/api/hub.html#methods).